app/models/manageiq/providers/ansible_tower/configuration_manager/playbook.rb

```diff
@@ -0,0 +1,4 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::Playbook < ConfigurationScript
+  belongs_to  :configuration_project, class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationSource"
+  has_many    :configuration_script,  class_name: "ManageIQ::Providers::AnsibleTower::ConfigurationManager::ConfigurationScript"
```

Comment:
These should also going info configuration_script as well, so you don't need the class_name.

Also configuration_project needs to be renamed.